src/theme/MDXComponents.js

```diff
@@ -1,11 +1,13 @@
 import MDXComponents from '@theme-original/MDXComponents';
 
 /* Components */
+import DocCardList from '@theme/DocCardList';
```

Comment:
@elof I moved this import into `MDXComponents.js` file, so you won't need to use the `import` statement in MDX files anymore.